packages/short-css-vars/index.test.js

```diff
@@ -72,12 +72,21 @@ describe('ShortCssVars', () => {
     });
 
     describe('ignore', () => {
+      it('ignores variable names via string', () => {
+        const input = ':root{--custom-var-one: red; --custom-var-two: 1.2rem}' +
+          '.my-class{ color: var(--custom-var-one, var(--custom-var-two))}';
+        const expected = ':root{--custom-var-one: red; --1tn4ykf: 1.2rem}' +
+          '.my-class{ color: var(--custom-var-one, var(--1tn4ykf))}';
+        const result = run(input, { ignore: 'one$' });
+        assume(result).equals(expected);
+      });
+
       it('ignores variable names via RegExp', () => {
         const input = ':root{--custom-var-one: red; --custom-var-two: 1.2rem}' +
           '.my-class{ color: var(--custom-var-one, var(--custom-var-two))}';
         const expected = ':root{--custom-var-one: red; --1tn4ykf: 1.2rem}' +
           '.my-class{ color: var(--custom-var-one, var(--1tn4ykf))}';
-        const result = run(input, { ignore: /one/ });
+        const result = run(input, { ignore: new RegExp('one$') });
```

Comment:
why this change? does `/one/` not work for some reason?

Reply:
I think I was just trying some things, I'll change it back to slashes